pom.xml

```diff
@@ -518,7 +522,9 @@
               <argument>run</argument>
               <argument>--pull=${cryostat.itest.pullImages}</argument>
               <argument>--pod=${cryostat.itest.podName}</argument>
-              <argument>--name=grafana-itest</argument>
+              <argument>--name=${cryostat.itest.grafana.containerName}</argument>
+              <argument>--health-cmd</argument>
+              <argument>wget -q --spider http://localhost:${cryostat.itest.grafana.port}</argument>
```

Comment:
Wondering why we used `wget` here?

Reply:
This container doesn't have `curl` in it, so this was the closest available equivalent.

`man wget`:
```
       --spider
           When invoked with this option, Wget will behave as a Web spider, which means that it will not download
           the pages, just check that they are there.  For example, you can use Wget to check your bookmarks:

                   wget --spider --force-html -i bookmarks.html

           This feature needs much more work for Wget to get close to the functionality of real web spiders.
```

Reply:
How about https://grafana.com/docs/grafana/latest/developers/http_api/other/#returns-health-information-about-grafana?

```bash
http :3000/api/health
HTTP/1.1 200 OK
Cache-Control: no-store
Content-Length: 70
Content-Type: application/json; charset=UTF-8
Date: Fri, 09 Jun 2023 21:41:15 GMT
X-Content-Type-Options: nosniff
X-Xss-Protection: 1; mode=block

{
    "commit": "4add91f03d",
    "database": "ok",
    "version": "9.4.7"
}

```